tang/.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
```

Comment:
Since you've set `root = true`, you should put this file to the **root** path of this repository, and benefit other contributors as a bonus.